integration_tests/e2e/induction/createLongQuestionSetInduction.cy.ts

```diff
@@ -166,6 +169,28 @@ context('Create a long question set Induction', () => {
     // Future Work Interest Types page is next
     Page.verifyOnPage(FutureWorkInterestTypesPage) //
       .hasBackLinkTo('/prisoners/A00001A/create-induction/previous-work-experience')
+      .submitPage() // submit the page without answering the question to trigger a validation error
+    Page.verifyOnPage(FutureWorkInterestTypesPage) //
+      .hasBackLinkTo('/prisoners/A00001A/create-induction/previous-work-experience')
+      .hasErrorCount(1)
+      .hasFieldInError('workInterestTypes')
+      .chooseWorkInterestType(WorkInterestTypeValue.OUTDOOR)
+      .chooseWorkInterestType(WorkInterestTypeValue.DRIVING)
+      .chooseWorkInterestType(WorkInterestTypeValue.OTHER)
+      .setWorkInterestTypesOther('Natural world')
+      .submitPage()
+
+    // Future Work Interest Roles page is next, with a field for each work interest type
+    Page.verifyOnPage(FutureWorkInterestRolesPage) //
```

Comment:
Work Interest Roles page doesnt have a validator, so no need to submit the page to trigger a validator error like we do on all the other pages in this test